components/compact-log-backup/src/storage.rs

```diff
@@ -535,6 +535,29 @@ impl LogFile {
     }
 }
 
+#[derive(derive_more::Deref, derive_more::DerefMut, Debug)]
+/// A migration with version and creator info.
+/// Preferring use this instead of directly create `Migration`.
+pub struct VersionedMigration(Migration);
+
+impl From<Migration> for VersionedMigration {
+    fn from(mut mig: Migration) -> Self {
+        mig.set_version(brpb::MigrationVersion::M1);
```

Comment:
Out of curiosity, is there a specific reason for the name "M1", as I think "V1" sounds more idiomatic.

Reply:
Because "V1" has already be used by another message in the same package, while protocol buffer requires every enum variant has a unique name in a package...